tools/psa/tfm/bin_utils/assemble.py

```diff
@@ -25,7 +25,7 @@
 import re
 import os
 import shutil
-from . import macro_parser
+import macro_parser
```

Comment:
This implicit relative import may cause issues in Python 3 depending on how this module is executed. We should keep the explicit relative import syntax.

Reply:
We need to make this change in TF-M.